app/views/hq/sponsors/edit.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Edit Partner #{@sponsor.name}, OSRA number #{@sponsor.osra_num}
+= render partial: 'form', locals: { sponsor: @sponsor, statuses: @statuses, sponsor_types: @sponsor_types, organizations: @organizations, branches: @branches, cities: @cities } 
```

Comment:
`_form.html.haml` is independent now. 
And a facade would be very helpful here.